VirtualReality/Widgets/qMRMLVirtualRealityHomeWidget.cxx

```diff
@@ -106,6 +113,57 @@ vtkMRMLVirtualRealityViewNode* qMRMLVirtualRealityHomeWidget::virtualRealityView
   return d->VirtualRealityViewNode;
 }
 
+//-----------------------------------------------------------------------------
+void qMRMLVirtualRealityHomeWidget::addModuleButton(QWidget* widget, QIcon* icon)
```

Comment:
Since you need to pass QIcon as not a pointer but as value, please change the signature of this function to
qMRMLVirtualRealityHomeWidget::addModuleButton(QWidget* widget, QIcon& icon)
(& instead of *)
This way your program won't crash if icon is nullptr.
That said please add a check for
if (!widget)
{
  qCritical().....
  return
}
Please try not to forget checking the inputs. Otherwise your program will crash!